### PR TITLE
prompt health/activity stats report table

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Add new prompt health table to the Activity Stats page.

## WHY
To show better data about how students are performing on prompts overall, rather than by rule.

## HOW
Mostly just display data from query that Peter Kong wrote on the backend.

### Screenshots
Screen Shot 2021-05-25 at 10 13 41 AM

### Notion Card Links
https://www.notion.so/quill/Frontend-New-Prompt-Health-Report-Table-ef555dee4d3c401398abf4bd40c826d1

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  YES
Have you deployed to Staging? | YES
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Design Review: If applicable, have you compared the coded design to the mockups? | YES

### CODE REVIEW

**Review by happythenewsad — APPROVED** [review]

**Review by anathomical — APPROVED** [review]
Looks good to me!

File: services/QuillLMS/client/app/bundles/Staff/components/comprehension/__tests__/activityStats.test.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
...system because\",\"num_final_attempt_optimal\":224,\"num_final_attempt_not_optimal\":0,\"avg_attempts_to_optimal\":2.6919642857142856,\"num_sessions_with_consecutive_repeated_rule\":208,\"num_sessions_with_non_consecutive_repeated_rule\":220,\"num_first_attempt_optimal\":180,\"num_first_attempt_not_optimal\":44},\"119\":{\"total_responses\":3176,\"session_count\":155,\"display_name\":\"A surge barrier in New York City could harm the local ecosystem, but\",\"num_final_attempt_optimal\":155,\"num_final_attempt_not_optimal\":0,\"avg_attempts_to_optimal\":2.6129032258064515,\"num_sessions_with_consecutive_repeated_rule\":139,\"num_sessions_with_non_consecutive_repeated_rule\":150,\"num_first_attempt_optimal\":135,\"num_first_attempt_not_optimal\":20},\"120\":{\"total_responses\":3291,\"session_count\":195,\"display_name\":\"A surge barrier in New York City could harm the local ecosystem, so\",\"num_final_attempt_optimal\":195,\"num_final_attempt_not_optimal\":0,\"avg_attempts_to_optimal\":2.2153846153846155,\"num_sessions_with_consecutive_repeated_rule\":182,\"num_sessions_with_non_consecutive_repeated_rule\":190,\"num_first_attempt_optimal\":161,\"num_first_attempt_not_optimal\":34}}"
```
Comment (eadams17) [inline services/QuillLMS/client/app/bundles/Staff/components/comprehension/__tests__/activityStats.test.tsx:9]: I think we could move this into `'../__mocks__/data';`

File: services/QuillLMS/client/app/bundles/Staff/components/comprehension/activityStats/activityStats.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+  debugger;
+
+  return(
+    <div className="activity-stats-container">
+      <h1>Activity Stats</h1>
+      {formattedRows && (<ReactTable
+        className="activity-stats-table"
+        columns={dataTableFields}
+        data={formattedRows ? formattedRows : []}
```
Comment (eadams17) [inline services/QuillLMS/client/app/bundles/Staff/components/comprehension/activityStats/activityStats.tsx]: could just make this `formattedRows || []`

**Review by eadams17 — APPROVED** [review]
LGTM!

**Pull request merged**